distribution/packages/build.gradle

```diff
@@ -48,7 +48,7 @@ buildscript {
     maven { url 'https://jitpack.io' }
   }
   dependencies {
-    classpath "com.github.breskeby:gradle-ospackage-plugin:ddb72a9922b934033827d48d296f7f3d470ac422"
+    classpath "com.github.breskeby:gradle-ospackage-plugin:98455c1"
```

Comment:
This is a snaspshot reference to the ospackage plugin containing the changes of this provided PR https://github.com/nebula-plugins/gradle-ospackage-plugin/pull/400